spec/Appendix C -- Schema Selectors.md

```diff
@@ -0,0 +1,168 @@
+# Appendix: Schema Selectors
+
+A Schema Selector is a human readable string that uniquely identifies an element defined in a GraphQL Schema.
+
+## Definition 
+
+SchemaSelector :
+  - TypeName FieldSelector?
+  - InterfaceName FieldSelector?
+  - EnumName EnumValueSelector?
+  - @ DirectiveName ArgumentSelector?
```

Comment:
Why is the `@` symbol needed here? If I am not mistaken, then from the point of view of the GraphQL type system the directive should have a unique name among all other types. What is the point of using an additional syntax element in the selector for the directive?

Reply:
Good question - I had considered this, and i'm certainly open to changing this.

This was a purely stylistic/aesthetics choice. Given the use cases, seeing "@private" in the output of some tooling would more obviously indicate "this relates to a directive" to me.

(I realize we didn't make any such distinction for enums or interfaces, but we're making up the rules here.)

There's an existing built in hint for directive syntax (the @ symbol), so using it here too to mirror.

Hope this makes sense!

Again, if folks feel strongly, I'm open to changing this?

Thanks!